### PR TITLE
add an option to insert html in footer

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

File: lib/dartdoc.dart [hunk 1]
```diff
@@ -29,8 +29,8 @@ const String NAME = 'dartdoc';
 const String VERSION = '0.0.1';
 
 /// Initialize and setup the generators
-List<Generator> initGenerators(String url) {
-  return [new HtmlGenerator(url)];
+List<Generator> initGenerators(String url, String footer) {
```
Comment (sethladd) [inline lib/dartdoc.dart]: should footer be a named optional param?

File: templates/_footer.html [hunk 2]
```diff
@@ -23,6 +23,8 @@
         prettyPrint();
     });
 </script>
+
+<!-- Footer Placeholder -->
```
Comment (sethladd) [inline templates/_footer.html]: maybe a note that says "do not remove"

File: lib/dartdoc.dart (hunk 1 above)
Comment (keertip) [inline lib/dartdoc.dart]: done.

File: templates/_footer.html (hunk 2 above)
Comment (keertip) [inline templates/_footer.html]: done.

**Pull request merged**